README.rst

```diff
@@ -121,6 +121,21 @@ We implement several other checks in order to maintain a consistent code style.
 
 in the repository.
 
+Citing
+------
+
+If you used Adaptive in a scientific work, please cite it as follows.
+
+.. code::
+
+    @misc{Nijholt2019,
+      doi = {10.5281/zenodo.1182437},
+      author = {Bas Nijholt and Joseph Weston and Jorn Hoofwijk and Anton Akhmerov},
+      title = {\textit{Adaptive}: Tools for adaptively sampling of mathematical functions in parallel},
```

Comment:
I've slightly rephrased it. I think it is better because otherwise `adaptive` appears twice in close procession.